redis_store.go

```diff
@@ -125,6 +125,29 @@ func (this *RedisStore) Subscribe(c chan []string, channel string) (*redis.Clien
 	return consumer, nil
 }
 
+func (this *RedisStore) Poll(c chan string, queue string) (*redis.Client, error) {
+	consumer := redis.New()
+	err := consumer.ConnectNonBlock(this.server, this.port)
+	if err != nil {
+		return nil, err
+	}
+
+	if _, err := consumer.Ping(); err != nil {
+		return nil, err
+	}
+
+	go func() {
+		for {
+			message, _ := consumer.LPop(queue)
+			c <- message
+
+			time.Sleep(time.Millisecond * 50)
```

Comment:
(derp) That makes sense. I was reading this backwards. Like he was reading from a channel